packages/paste-website/src/pages/foundations/content/content-checklist/index.mdx

```diff
@@ -38,60 +38,32 @@ export const getStaticProps = async () => {
 
 <content>
 
-## Get content support
-
-### Training and resources
-
-Look out for information about content training sessions via email and sign up!
-
-Check out the latest resource: [Content for UX](https://docs.google.com/document/d/1nerpH9xhsG6obgl-nKWbzwJFQva5ZU6VdNgCRJx-zVY/edit?usp=sharing).
-This contains the key takeaways from the training session, action points, and tools for you to use.
-
-### Quick self-serve checklist
-
 How do you know if your content is "done" or "good"?
 
-#### The tone is appropriate to the scenario.
-
-- A stressful scenario is managed empathetically, with a knowledgeable and encouraging tone. Reassure and inform the user clearly.
-- A win or success is managed genuinely, with a positive and encouraging tone. Celebrate with the user genuinely.
-- A repeated interaction is managed efficiently, with a positive and conversational tone. Support the user's flow by reducing friction.
-- The content sounds like a frinedly specialist.
+**The content meets the user's immediate needs.**
+- The user can understand the context, the goal, the impact of any actions or decisions they’ll make, and knows what happens next.
 
-#### The content meets the user's immediate needs.
-
-- The user has enough information to understand the context and proposition, the impact of an action/decision, and clarity over what happens next.
-
-#### The words and structure are as simple as they can be.
-
-- You've replaced jargon with a simpler term or explained technical language or acronyms.
+**The words and structure are as simple as they can be.**
 - Sentences are short.
+- You've eliminated jargon where possible, and explained technical language if it’s required.
 
-#### The message is clear and unambiguous.
-
+**The message is clear and unambiguous.**
 - Your content can only be interpreted one way.
 - You've replaced any vague or imprecise language with precise, clear terms.
 
-#### You've checked it. Twice.
-
-- Capitals, punctuation, and spelling all align with this guide.
-- Tip: Read it out loud to yourself. If it doesn't "sound right", consider changing the wording or phrasing.
+**The tone is appropriate to the scenario.**
+- Stressful scenario: Content provides factual information and informs the user about what they can do next. Content tone is empathetic and reassuring.
+- Successful situation: Content celebrates the user. Content tone is positive and encouraging.
+- Repeated interaction: Content reduces friction and anticipates user needs. Content tone is positive and conversational.
+- Content generally sounds like a friendly specialist.
```

Comment:
I feel like this section is super helpful, and could be formatted into a table or some other way to call more attention to it.